lmdeploy/model.py

```diff
@@ -219,7 +226,7 @@ def messages2prompt(self, messages, sequence_start=True, **kwargs):
                 ret += f'{self.system}{self.meta_instruction}{self.eosys}'
         for message in messages:
             role = message['role']
-            content = message['content']
+            content = get_text(message['content'])
```

Comment:
Is it for the GPT4V messages?

Reply:
the content part could be str or [ChatCompletionContentPartTextParam](https://github.com/openai/openai-python/blob/main/src/openai/types/chat/chat_completion_content_part_text_param.py)